### PR TITLE
Emails: resurrect template notification

### PR DESCRIPTION
**Pull request by markelog**
* Phantomjs (oh yeah, there is another dev dep phantom :-) was failing for
  the generation of the html templates so I had to update the dependencies
  in order to fix it. While doing that I update the scripts field and docs
  for it as well. yarn.lock is included

* Move splitting of the emails to separate helper function, since more services
  coming up that would need to use this functionality

* Add support for enterprise specific email letters. Probably could
  be done in the better way, but it's not a priority right now

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,10 @@ awsconfig
 /public/views/index.html
 /public/views/error.html
 /emails/dist
+
+# Enterprise emails
+/emails/templates/for-enterprise-*
```
Comment (torkelo) [inline .gitignore]: Think a folder named enterprise that we can symlink in where all enterprise emails would be preferable but not sure if there is some problem with that. 

The emails get built and checked in currently in this folder, so we would need an enterprise folder ignored (and symlinked) here as well:

https://github.com/grafana/grafana/tree/master/public/emails

If a folder is tricky for some reason then a prefix like `enterprise_template-name.html` is best the other files already use underscore.

**Review by torkelo — COMMENTED** [review]
Looks good, added some comments regarding the ignore patterns and the fact that emails get built and checked in under public as well

File: .gitignore (hunk 1 above)
Comment (markelog) [reply .gitignore]: > Think a folder named enterprise that we can symlink in where all enterprise emails would be preferable but not sure if there is some problem with that.

Yeah, I was thinking the same thing! But I didn't want to deal with refactoring of the grunt config for it, was planning to create an issue for that

Comment (markelog) [reply .gitignore]: > The emails get built and checked in currently in this folder, so we would need an enterprise folder ignored (and symlinked) here as well:

> https://github.com/grafana/grafana/tree/master/public/emails

> If a folder is tricky for some reason then a prefix like enterprise_template-name.html is best the other files already use underscore.

Done!

**Review by xlson — APPROVED** [review]

**Pull request merged**